lambda/events/index.ts

```diff
@@ -1,8 +1,30 @@
-import { SQSEvent } from 'aws-lambda';
+import { SQSEvent, SQSBatchResponse, SQSBatchItemFailure } from 'aws-lambda';
 import * as Sentry from '@sentry/serverless';
+import { handlers } from './handlers';
 
-function processor(event: SQSEvent) {
-  console.log(`Events Lambda invoked with: ${JSON.stringify(event)}`);
+/**
+ * The main handler function which will be wrapped by Sentry prior to export
+ * @param event
+ * @returns
+ */
+export async function processor(event: SQSEvent): Promise<SQSBatchResponse> {
+  const batchFailures: SQSBatchItemFailure[] = [];
+  for await (const record of event.Records) {
+    try {
+      const message = JSON.parse(JSON.parse(record.body).Message);
+      if (handlers[message['detail-type']] == null) {
+        throw new Error(
+          `Unable to retrieve handler for detail-type='${message['detail-type']}'`
+        );
+      }
+      await handlers[message['detail-type']](record);
+    } catch (error) {
+      console.log(error);
```

Comment:
just noting the `console.log` here (these def help in debugging once in infrastructure - not saying to remove it)